x-pack/test/functional/apps/ml/data_frame_analytics/outlier_detection_creation.ts

```diff
@@ -58,8 +58,8 @@ export default function ({ getService }: FtrProviderContext) {
             { key: '#F5F7FA', value: 2 },
             { key: '#D3DAE6', value: 1 },
             // scatterplot circles
-            { key: '#54B399', value: 1 },
-            { key: '#54B39A', value: 1 },
+            { key: '#69707D', value: 1 },
+            { key: '#98A1B3', value: 1 },
```

Comment:
@pheyos We might want to consider another key for these values, but I hope this works

Reply:
@spalger I'm working on an update to this code in #91473, will revisit there.